jenkins/public_tests_all_old.Jenkinsfile

```diff
@@ -0,0 +1,220 @@
+pipeline {
+  agent any;
+  environment {
+    MAKE_ISSUE = 1;
+  }
+  options {
+    copyArtifactPermission('${JOB_NAME},'+env.BRANCH_NAME);
+  }
+  stages {
+
+    stage('Local Build') {
+      agent any;
+      steps {
+        sh "./build_openroad.sh --local";
+        stash name: "install", includes: "tools/install/**";
+      }
+      post {
+        always {
+          catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
+            archiveArtifacts artifacts: "build_openroad.log";
+          }
+        }
+      }
+    }
+
+    stage('Tests') {
+      matrix {
+        axes {
+          axis {
+            name 'TEST_SLUG';
+            values "docker build",
+                   "aes asap7",
```

Comment:
Do you know if it is possible to read these values form one or multiple files?

Reply:
there's a possibility to define them as global variables configured in jenkins. There's another option to define a function in jenkins-ci that returns the list of test slugs.